pkg/cloudprovider/vagrant/vagrant.go

```diff
@@ -145,6 +152,11 @@ func (v *VagrantCloud) ExternalID(instance string) (string, error) {
 	return minion.IP, nil
 }
 
+// InstanceID returns the cloud provider ID of the specified instance.
+func (v *VagrantCloud) InstanceID(instance string) (string, error) {
+	return "", nil
```

Comment:
I don't think we need an extra "/" here. We know that the machines can only be local.